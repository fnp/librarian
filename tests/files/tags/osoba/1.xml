<naglowek_akt>AKT PIERWSZY</naglowek_akt>

<didaskalia>
Pokój w zamku
<osoba>Cześnika</osoba>,
drzwi na prawo, lewo i w środku, stoły, krzesła etc., gitara angielska na ścianie.
</didaskalia>

<naglowek_scena>SCENA PIERWSZA</naglowek_scena>

<didaskalia>

<osoba>Cześnik</osoba>,
<osoba>Dyndalski</osoba>

</didaskalia>
