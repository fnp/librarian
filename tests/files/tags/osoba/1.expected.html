<div id="book-text">
<div id="toc">
<h2>Spis treści</h2>
<ol>
<a href="#s1" class="anchor">AKT PIERWSZY</a><li><ol>
<a href="#s2" class="anchor">SCENA PIERWSZA</a><li>
</ol></li>
</ol>
</div>
<h2 class="wl" id="s1">AKT PIERWSZY</h2>
<div class="didaskalia" id="i1">
Pokój w zamku
<em class="person">Cześnika</em>,
drzwi na prawo, lewo i w środku, stoły, krzesła etc., gitara angielska na ścianie.
</div>
<h3 id="s2">SCENA PIERWSZA</h3>
<div class="didaskalia" id="i2">

<em class="person">Cześnik</em>,
<em class="person">Dyndalski</em>

</div>
</div>
